pkg/activator/handler/filtering_handler_test.go

```diff
@@ -34,13 +34,13 @@ func TestFilteringHandler(t *testing.T) {
 	},
 		{
 			label:          "filter a request containing retry header",
-			headers:        http.Header{activator.ResponseCountHTTPHeader: {"4"}},
+			headers:        http.Header{http.CanonicalHeaderKey(activator.RequestCountHTTPHeader): {"4"}},
 			passed:         false,
 			expectedStatus: http.StatusServiceUnavailable,
 		},
 		{
 			label:          "forward a request containing empty retry header",
-			headers:        http.Header{activator.ResponseCountHTTPHeader: {""}},
+			headers:        http.Header{http.CanonicalHeaderKey(activator.RequestCountHTTPHeader): {""}},
```

Comment:
Is the constant not yet already canonicalized? If not, should we?

Reply:
Of the existing headers two were lower case and one was canonicalized; I went with the dominate style. Happy to change it to whichever style is preferred.